pkg/errors/errors.go

```diff
@@ -107,9 +107,12 @@ var (
 	// ContextIsNotOktetoCluster raised if the cluster connected is not managed by okteto
 	ErrContextIsNotOktetoCluster = fmt.Errorf("this command is only available on Okteto Cloud or Okteto Enterprise")
 
-	// ErrTokenFlagNeeded is raised when the command is executed from inside a pod
+	// ErrTokenFlagNeeded is raised when the command is executed from inside a pod from a ctx command
 	ErrTokenFlagNeeded = fmt.Errorf("this command is not supported without the '--token' flag from inside a container")
 
+	// ErrTokenEnvVarNeeded is raised when the command is executed from inside a pod from a non ctx command
+	ErrTokenEnvVarNeeded = fmt.Errorf("this command is not supported without the 'OKTETO_TOKEN' environment variable set from inside a container")
```

Comment:
"The OKTETO_TOKEN environment variable is required when running this command from within a container."

Reply:
@jmacelroy does that error message sounds alright?